File: packages/python/mythos_sdk/report_usage.py
from .api_client import meter_session


async def report_usage(
    jti: str,
    credits: int,
    reason: str | None = None,
    *,
    idempotency_key: str | None = None,
) -> None:
    await meter_session(jti, credits, reason, charge_id=idempotency_key)

File: packages/node/src/reportUsage.ts
import { meterSession } from './api-client';

export async function reportUsage(
  jti: string,
  opts: { credits: number; reason?: string; chargeId?: string },
): Promise<void> {
  await meterSession(jti, opts.credits, opts.reason, opts.chargeId);
}
